docs/notebooks/pulse_efficient_transpilation_qsim.ipynb

```diff
@@ -0,0 +1,765 @@
+      "text/plain": [
+       "<Figure size 1039.79x200.667 with 1 Axes>"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# transpile into RZX gates\n",
+    "rzx_circ = transpile(qc, basis_gates=['rzx', 'x', 'sx', 'rz'])\n",
+    "print(f'two-qubit circuit depth = {rzx_circ.depth(lambda x: x[0].num_qubits == 2)}')\n",
+    "rzx_circ.draw('mpl', fold=-1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The RZX transpilation results in three `RZXGate`s, which effectively shortens the two-qubit circuit depth by half compared with the CNOT transpilation! In fact, this is a general result: any 2-qubit unitary can be represented in the `RZXGate` decomposition with at most three `RZXGate` via the so-called KAK decomposition. \n",
+    "\n",
+    "Notice that in the `RZXGate` transpiled circuit, the `RZXGate`s are parameterized by the angle `Jt.` To properly implement the gate, we need to instruct the hardware on the proper pulse schedule. This is done by creating a customized [pulse gate](https://qiskit.org/documentation/tutorials/circuits_advanced/05_pulse_gates.html) by attaching a stretched pulse schedule to the `RZXGate` based on the desired entangling angle. \n",
+    "\n",
+    "To show how to do this, we first pick a backend to transpile the circuit to and assign values to the circuit paramters."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 33,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "backend = service.get_backend('ibmq_mumbai')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# TODO: assign parameters by name instead of order\n",
```

Comment:
Remove TODO comment

Reply:
TODO comment removed in the new version.